atlas/packages/app/src/shared/components/Button/Button.style.tsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react'
+import { css } from '@emotion/core'
+import styled from '@emotion/styled'
+import { BlockIcon } from '@/shared/icons'
+import { typography, colors, spacing } from '../../theme'
+
+export type ButtonStyleProps = Partial<{
+  variant: 'primary' | 'secondary' | 'tertiary'
+  full: boolean
+  size: 'regular' | 'small' | 'smaller'
+  children: React.ReactNode
+  onClick: (e: React.MouseEvent<HTMLButtonElement>) => void
+  disabled: boolean
+  hoverable: boolean
```

Comment:
What does `hoverable` mean? Is that related to whether the button should have a hover raise animation? If so, I think we should find a more appropriate name, e.g. `animated` or something like that. Every button is hoverable, even when disabled :P

Reply:
Good point, `animated` is more descriptive, I replaced all mentions of `hoverable`.